.github/workflows/lint.yml

```
@@ -0,0 +1,19 @@
+name: Lint
+
+on:
+  push
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  lint:
+    runs-on: ubuntu-22.04
```

Please modify version specification to `ubuntu-latest` to avoid breakage due to version updates.
```suggestion
    runs-on: ubuntu-latest
```